astrocut/cutouts.py

```diff
@@ -279,7 +279,7 @@ def fits_cut(input_files, coordinates, cutout_size, correct_wcs=False, extension
         with fits.open(in_fle, mode='denywrite', memmap=True, fsspec_kwargs=fsspec_kwargs) as hdulist:
 
             # Sorting out which extension(s) to cutout
-            all_inds = np.where([x.is_image and (x.data is not None) for x in hdulist])[0]
+            all_inds = np.where([x.is_image and (x.section.shape != ()) for x in hdulist])[0]
```

Comment:
For this line and 540, what was the rationale for changing this comparison? Could we use `x.section is not None` or `not x.size`?

Reply:
Hi Sam! This is the line I wondered about most myself. I did some testing with `.section` and found that `x.section` is not None, even for empty primary HDUs.  So I know a simple `x.section is not None` won't work.  The `x.section.shape != ()` weeds out the empty primary HDU.  But `x.size != 0` would also eliminate the empty primary HDU.

It is necessary to include the `x.is_image` test to avoid selecting table HDUs.  I tested a file with a BinTableHDU and found that (1) `x.size` is not zero, (2) `x.is_image` is False, and (3) `x.section` is not defined (raises an AttributeError).  

So that's how I came up with my code. I think either of these could work:
```python
x.is_image and (x.section.shape != ())
```
or
```python
x.is_image and (x.size > 0)
```
I do kinda like the `x.size` version of that better.  So I'd be happy with that change.

Reply:
I just made that change, see what you think.

Reply:
Looks good to me - I committed those changes.